odp/identity/templates/base.html

```diff
@@ -82,5 +101,9 @@
 
```

Comment:
Add a footer for sadco. This can be exactly the same as for nccrd, but it should be in its own elif block.